cardano-cli/test/cardano-cli-golden/files/golden/governance/hash/foo.script.hash

```diff
@@ -1 +1 @@
-2a7a40032759083907e43e145b770b467cfc0ddbc3c1266bc2c01b40
\ No newline at end of file
+7dde1be4d1a2f617ac343dd7ef0388cf497c19ba8dab653a5b9eb1e2
```

Comment:
What's the reason for this hash change?

Reply:
I think that's an expected result from: https://github.com/IntersectMBO/cardano-api/pull/452